src/uct/tcp/tcp_sockcm_ep.h

```diff
@@ -26,8 +26,9 @@ typedef enum uct_tcp_sockcm_ep_state {
                                                                     (debug flag) */
     UCT_TCP_SOCKCM_EP_PACK_CB_FAILED              = UCS_BIT(15), /* the upper layer's priv_pack_cb failed */
     UCT_TCP_SOCKCM_EP_SERVER_REJECT_CALLED        = UCS_BIT(16), /* ep on the server called reject API call */
-    UCT_TCP_SOCKCM_EP_SERVER_REJECT_SENT          = UCS_BIT(17)  /* ep on the server sent the reject message to the client */
-
+    UCT_TCP_SOCKCM_EP_SERVER_REJECT_SENT          = UCS_BIT(17), /* ep on the server sent the reject message to the client */
+    UCT_TCP_SOCKCM_EP_RESOLVE_CB_FAILED           = UCS_BIT(18), /* the upper layer's resolve_cb failed */
+    UCT_TCP_SOCKCM_EP_RESOLVED                    = UCS_BIT(19)  /* resolve_cb returned UCS_OK */
```

Comment:
maybe UCT_TCP_SOCKCM_EP_RESOLVE_CB_INVOKED?
(would also need to change it's usage a bit)
to align with other flags